srcpkgs/pyradio/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'pyradio'
 pkgname=pyradio
-version=0.8.9.9
-revision=2
+version=0.8.9.14
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-requests python3-dnspython"
+depends="python3-requests python3-dnspython python3-psutil"
 short_desc="Curses based internet radio player"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="http://www.coderholic.com/pyradio/"
 changelog="https://raw.githubusercontent.com/coderholic/pyradio/master/ChangeLog"
```

Comment:
404: Not Found

Reply:
Thanks.
Updated it, committed and squashed.